Gulpfile.js

```diff
@@ -16,7 +16,8 @@ var gulp = require('gulp'),
     connect = require('gulp-connect'),
     svg2png = require('gulp-svg2png'),
     webpack = require('webpack-stream'),
-    zip = require('gulp-zip')
+    zip = require('gulp-zip'),
+    exec = require('child_process').exec
```

Comment:
Was adding this for making `gulp test` work? There doesn't seem to be a test task. Else do the `gulp test` references in other files in this PR need to be updated?